### ISSUE TITLE
Update a11y for Contact VA page

### ISSUE THREAD

**Issue by esther-k** [department-of-veterans-affairs/va.gov-team#15089]
## Dev task: 
1. Update the a11y for ticket #14015 (criteria below)
2. Related bug to complete in-tandem: [#16125](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/16125) 
## Designs:
https://adhoc.invisionapp.com/share/YUYRT9BAPSB#/screens/432493860

## Accessibility Acceptance Criteria
1. Everything on the page should be read by the screen reader.
     - If anything is being conveyed visually only (buttons, menu items, list, tables, images, headings, forms, etc) the component type needs to be announced by the screenreader. (Include name (title), role, value)
2. Each page should support text resizing based on the OS font scaling settings without loss of content, content overlap, or horizontal scroll.
3. Each touch target must be at minimum 44px by 44px
4. The app needs to support operability by tap (no gestures) or voice control only. In cases where gestures occur, e.g. scroll, ensure these simple gestures are operable through the OS’ built-in capabilities.  
5. All elements on a page should be able to be traversed by an auxiliary keyboard. As a user uses the keyboard to navigate a page, its focus element should be clear.
6. When a page opens, the first section or element should automatically be placed in focus
7. When a page opens, the title page should be announced

## Accessibility Implementation Guidelines 
- Back button in top left
     - accessibilityLabel= “Back”
     - accessibilityHint= “Go back to home”
     - accessibilityRole= “button”
- Mark "Contact VA" as Header 1
- Talk to the veterans Crisis Line now CTA
     - accessibilityLabel= "Talk to the Veterans Crisis Line now”
     - accessibilityHint= “Go to Veterans Crisis Line”
     - accessibilityRole= “button”

**Contact Links**
- Icons
     - Title= “ Phone Icon”
     - accessibilityRole= “image”
 - Contact Resource
     - Title= Name of link (e.g. 844-698-2311)
     - accessibilityHint= “Dials this number via your device’s call function"
     - accessibilityRole= “link”

**Comment by digitalken** [comment]
Added to TestRail.

**Comment by keli13** [comment]
Will being ignoring the accessibilityHint for the back button as there will be an update in a separate [ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/17185)

**Comment by esther-k** [comment]
**12/10**

Pass on Android. Need to retest on iOS. Keeping in QA
